cli/azd/cmd/infra_create.go

```diff
@@ -22,6 +21,7 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"github.com/theckman/yacspin"
+	"github.com/vhvb1989/survey/v2"
```

Comment:
Instead of updating the import path here, can you look into using the [replace](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive) feature of `go.mod` so that we keep the original import path but point it at your fork for now?